src/Microsoft.AspNetCore.Http.Abstractions/project.json

```diff
@@ -11,11 +11,13 @@
   },
   "dependencies": {
     "Microsoft.AspNetCore.Http.Features": "1.0.0-*",
+    "Microsoft.Extensions.FileProviders.Abstractions": "1.0.0-*",
```

Comment:
It's not just FileProviders, it's FileProviders.Abstractions. In my head, it made sense that one abstraction package depends on another :smile: